packages/@aws-cdk/aws-appmesh/lib/tls-validation.ts

```diff
@@ -1,33 +1,36 @@
 import * as acmpca from '@aws-cdk/aws-acmpca';
-import { CfnVirtualGateway, CfnVirtualNode } from './appmesh.generated';
+import { CfnVirtualNode } from './appmesh.generated';
 
 // keep this import separate from other imports to reduce chance for merge conflicts with v2-main
 // eslint-disable-next-line no-duplicate-imports, import/order
 import { Construct } from '@aws-cdk/core';
 
 /**
- * Represents the properties needed to define TLS validation context
+ * Represents the properties needed to define TLS Validation context
  */
 export interface TlsValidation {
+  /**
+   * Represents the subject alternative names (SANs) secured by the certificate.
+   * SANs must be in the FQDN or URI format.
+   *
+   * @default - the Envoy proxy for that node doesn't verify the SAN on a peer client certificate.
+   */
+  readonly subjectAlternativeNames?: SubjectiveAlternativeNames;
```

Comment:
Add this new property below the `trust` property, since that one is required, and this is optional.

Reply:
Thanks for adding the detail on the change! This is addressed.